plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -765,6 +768,8 @@ void producer_plugin::set_program_options(
           "Number of worker threads in producer thread pool")
          ("snapshots-dir", bpo::value<bfs::path>()->default_value("snapshots"),
           "the location of the snapshots directory (absolute path or relative to application data dir)")
+         ("max-read-only-transaction-time", bpo::value<int32_t>()->default_value(150),
+          "Limits the maximum time (in milliseconds) that is allowed a pushed read-only transaction's code to execute before being considered invalid")
```

Comment:
`Limits the maximum time (in milliseconds) a read-only transaction can execute before being considered invalid`